fluentd/configs.d/openshift/output-es-ops-config.conf

```diff
@@ -1,6 +1,7 @@
     <store>
       @type elasticsearch
       @id elasticsearch-ops
+      @log_level "#{ENV['ES_OPS_LOG_LEVEL'] || 'warn'}"
```

Comment:
```
@log_level "#{ENV['ES_OPS_LOG_LEVEL'] || ENV['ES_LOG_LEVEL'] || 'warn'}"
```
that way, you can just set `ES_LOG_LEVEL` and it will apply to both ops and non-ops, or you can set `ES_OPS_LOG_LEVEL` separately